docs/pages/docs/form/checkbox.mdx

```diff
@@ -6,12 +6,18 @@ import LinkBadgeGroup from "../../../components/link-badge/link-badge-group.tsx"
 # Checkbox
 A control that allows the user to toggle between checked and not checked.
 
-For touch devices, a [switch](/docs/form/switch) is generally recommended over a checkbox.
+
 
 <LinkBadgeGroup>
     <LinkBadge label="API Reference" href="https://pub.dev/documentation/forui/latest/forui.widgets.checkbox/forui.widgets.checkbox-library.html"/>
 </LinkBadgeGroup>
 
+<div className="pb-5">
+    <Callout type="info">
+        For touch devices, a [switch](/docs/form/switch) is generally recommended over this.
+    </Callout>
+</div>
+
 <div className="pb-5">
```

Comment:
I think you can nest both callouts together. The gap between both callouts look awfully big